package.json

```diff
@@ -89,6 +89,10 @@
           "default": true,
           "markdownDescription": "Specifies whether to enable Stripe telemetry (even if enabled still abides by the overall `#telemetry.enableTelemetry#` setting)",
           "scope": "window"
+        },
+        "stripe.java.home": {
+          "type": "string", 
+          "description": "Java home path of the JDK for Java language support"
```

Comment:
can we scope this to "machine" like the CLI path? This ensures that the setting can't be just set by a repo.

Reply:
sure!